apis/dialapi/dialapi.yaml

```diff
@@ -427,8 +427,132 @@ paths:
             to contact us. In the meantime, try refreshing.
           content: {}
       deprecated: false
-      security: 
+      security:
         - bearer: []
+  /dialcode/v2/read:
+    post:
+      tags:
+        - DIAL APIs
+      summary: Get DIAL Code V2
+      description: |-
+        This API is associated with fetching/reading the DIAL code with context information
+          - The endpoint for **Get DIAL code V2** is `/dialcode/v2/read/{dialcode}`
+          - It points to knowledge-mw-service - `/v2/dialcode/read/{dialcode}`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty."
+      operationId: ReadPost
+      parameters:
+        - name: X-Channel-ID
+          in: header
+          description: It the Unique ID to identify the root organization to which the
```

Comment:
Please change it to "It's" in the description.

Reply:
done